tests/break-calls/type_args/jsfmt.spec.js

```diff
@@ -0,0 +1,2 @@
+run_spec(__dirname, ["flow", "typescript"]);
+run_spec(__dirname, ["babel"]);
```

Comment:
Please remove this line. `babel` does not and will not support this syntax. So what's the point?

Reply:
I think it would be good for it to be known and remain known how the Babel parser would handle the input, even if it is not designed to do anything special.

Reply:
Good for what? How does it help anything? Why not add also `markdown` and `html` to this test then?

Reply:
Show what should Prettier do then if a user puts this input when using the Babel parser?

Reply:
Are we going to test every possible input and parser combination?